Documentation/data-collection.md

```diff
@@ -225,6 +225,17 @@ data:
     # (workloads-team, @openshift/openshift-group-b) cluster_master_schedulable reports whether mastersSchedulable=true in
     # the scheduler operator. This value is a boolean 0|1
     - '{__name__="cluster_master_schedulable"}'
+    # (codeready workspaces) The number of workspaces with a given status STARTING|STOPPED|RUNNING|STOPPING. Type 'gauge'.
```

Comment:
Please add a team (github alias) owning these metrics.

Reply:
the repo is under https://github.com/redhat-developer/codeready-workspaces I do not think we have some team alias

Reply:
We need some contact team internally who are responsible for those metrics.

Reply:
sure, but can it be an internal slack channel reference?

Reply:
No, its for pinging on github. Ideally a team within openshift org or individual person we can ping when we change/rename/remove metrics.

Reply:
@vparfonov @ibuziuk please add a team alias in the openshift organization where we can ping people in case we run into troubles with those metrics.